### DISCUSSION TITLE
热重载

### DISCUSSION THREAD

**Discussion by tianzhentech** [MetaCubeX/mihomo#1541] [Q&A]
api可以热重载配置文件里的节点密码，端口或者uuid等信息吗，而不必重启内核？有这样的功能吗

**Comment by xiandu999-yu** [comment]
![image](https://github.com/user-attachments/assets/9ae3cbcc-ca00-494a-a913-904cd56e11ce)
![image](https://github.com/user-attachments/assets/681aaabf-3bfb-4dc3-b8ad-858da49e7cbf)

**Reply by tianzhentech** [reply]
感谢佬，忘记这个了，用api也可以是吗，但是我用api一直没成功。
